<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSchedConf</class>
 <widget class="QWidget" name="VSchedConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCVR video</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../velibui.qrc">
    <normaloff>:/action/resources/libuivsc32.png</normaloff>:/action/resources/libuivsc32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pbOff">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/libuibutton.png);}
QPushButton:hover {
	background-image: url(:/action/resources/libuibuttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/libuibutton.png);}
	font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>Off</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSched">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/libuibutton.png);}
QPushButton:hover {
	background-image: url(:/action/resources/libuibuttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/libuibutton.png);}
	font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>Sched</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbEvent">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/libuibutton.png);}
QPushButton:hover {
	background-image: url(:/action/resources/libuibuttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/libuibutton.png);}
	font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>Event</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbAllDay">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/libuibutton.png);}
QPushButton:hover {
	background-image: url(:/action/resources/libuibuttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/libuibutton.png);}
	font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>All Day</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbWorkDay">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/libuibutton.png);}
QPushButton:hover {
	background-image: url(:/action/resources/libuibuttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/libuibutton.png);}
	font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>Work Day</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClear">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { 
border:none;
color:white;
background-image: url(:/action/resources/libuibutton.png);}
QPushButton:hover {
	background-image: url(:/action/resources/libuibuttonpress.png);}
QPushButton:pressed {
	background-image: url(:/action/resources/libuibutton.png);}
	font: 8pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="VSchedTable" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>720</width>
       <height>210</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VSchedTable</class>
   <extends>QWidget</extends>
   <header>vschedtable.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../velibui.qrc"/>
 </resources>
 <connections/>
</ui>
